infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/project/OpenShiftProjectFactory.java

```diff
@@ -44,4 +44,9 @@ public OpenShiftProject create(String workspaceId) throws InfrastructureExceptio
     final String projectName = isNullOrEmpty(this.projectName) ? workspaceId : this.projectName;
     return new OpenShiftProject(clientFactory, projectName, workspaceId);
   }
+
+  public OpenShiftProject create(String workspaceId, String projectName)
```

Comment:
Can you add javadocs with clarification why 2 methods exist?

Reply:
Added